### PR TITLE
[CFM-1274] -reconfigure og_menu.

### PR DESCRIPTION
**Pull request by ybaras**
[CFM-1274](https://issuetracker.amplexor.com/jira/browse/CFM-1274)

### Dependent on #1268

### CODE REVIEW

**Comment by ybaras** [comment]
603e4a2f-bc1a-415c-b809-6dc227c52e5f

**Comment by ordavidil** [comment]
@ybaras 
please re-roll again with  `features/cfm-1512`

File: project/profiles/capacity4more/capacity4more.info [hunk 1]
```diff
@@ -58,7 +58,6 @@ dependencies[] = og_ui
 dependencies[] = og_purl
 dependencies[] = og_vocab
 dependencies[] = og_menu
```
Comment (ordavidil) [inline project/profiles/capacity4more/capacity4more.info]: @ybaras 
do we still need this module?

Comment (ybaras) [inline project/profiles/capacity4more/capacity4more.info]: Yes, we do.
We don't need `og_invite_people` but that's handled in #1142 .

**Pull request merged**